docs/IKOS.md

```diff
@@ -0,0 +1,50 @@
+
+# IKOS Integration
+
+IKOS is a static analysis tool that performs interprocedural analysis on C/C++ code.
+
+Make sure you are in the Space ROS Docker container before running the following commands.
+
+## Running an IKOS Scan
+
+IKOS uses special compiler and linker settings in order to instrument and analyze binaries.
+To run an IKOS scan on all of the Space ROS test binaries (which will take a very long time), run the following command at the root of the Space ROS workspace:
+
+```
+spaceros-user@d10d85c68f0e:/opt/spaceros$ CC="ikos-scan-cc" CXX="ikos-scan-c++" LD="ikos-scan-cc" colcon build --build-base build_ikos --install-base install_ikos --cmake-args -DSECURITY=ON -DINSTALL_EXAMPLES=OFF -DCMAKE_EXPORT_COMPILE_COMMANDS=ON --no-warn-unused-cli
```

Comment:
Yeah, if we figure out where to put the source we should update these with the merge install option, too. Assuming that will work?

Reply:
Yes, that should work without a problem, we will need to slilghtly modify the command statements